spec/features/saml/saml_spec.rb

```diff
@@ -493,12 +493,14 @@
 
         expect(current_url).to eq idv_url
 
-        click_on 'Continue'
+        click_on 'Yes'
 
         expect(page).to have_content(t('idv.form.first_name'))
 
         fill_out_idv_form_ok
-        click_button 'Continue'
+        click_button 'Continue verifying'
+        fill_out_financial_form_ok
+        click_button 'Continue verifying'
```

Comment:
agreed. We have a bunch of hard-coded text all over our specs, I think, that we ought to ensure are localized. I'll add a separate issue cause it's not (I don;t think) limited to these specs.